.travis.yml

```diff
@@ -17,5 +17,5 @@ install:
   - travis_retry python setup.py dev
 script: py.test --runslow
 after_success:
-  - coveralls
+  - coveralls --config_file=.coveragerc_nogpu
```

Comment:
Why not pass `--cov-config=.coveragerc-nogpu` to pytest instead?  This way the report at the end of the Travis run would be in line with what coveralls sees.